### ISSUE TITLE
improve focus indicators for buttons

### ISSUE THREAD

**Issue by cseppan** [USEPA-OAQPS/emf#98]
When tabbing through the interface, Java has a default highlight for buttons with a thin blue light around the button's text. This doesn't meet the accessibility contrast guidelines. The Sort-Filter-Select toolbar buttons are particularly bad since the highlight blends into the button's icon.

Need to see if there are any options for changing how buttons are highlighted when they have focus.

**Comment by ddelvecchio** [comment]
Initial button contrast (from background color to focus label:
![image](https://user-images.githubusercontent.com/7310150/146311149-48a8f82a-5032-4102-bd59-5448db3a23da.png)

**Comment by ddelvecchio** [comment]
In order to change the focus lablel to the correct contrast color ratio, the ration was increased to 7:1 which meets both WCAG AA and WCAG AAA.

The following line of code was added to the EmfClient.java run method.  This will globably apply to the whole application the new button focus color contrast.  This change affects all buttons and the sort/filter panels for the entire application.

        UIManager.put("Button.focus", new Color(51, 82, 107));
